DAQ/src/StrawAndCaloDigisFromFragments_module.cc

```diff
@@ -457,7 +458,7 @@ produce( Event & event )
 
   }
 
-  event.put(std::unique_ptr<EventNumber_t>(new EventNumber_t( eventNumber )));
+  //  event.put(std::unique_ptr<EventNumber_t>(new EventNumber_t( eventNumber )));
```

Comment:
Is there a reason this is commented and not removed?  Is it an optional data product?

Reply:
My point was that you should delete the line not just comment it out, unless this is a future option, in which case it should be controlled but fcl.